Chapter-4-applying-tactical-domain-driven-design/README.adoc

```diff
@@ -88,13 +88,38 @@ That's it! You should now be able to run the application using either one of the
 
 === Building and debugging code in Rider IDE
 
-Before you build or debug code in `Rider` environment, you first have to configure artifactory to allow to download packages for `Common` which is a part of this repository:
+Before you build or debug code in `Rider` or `Visual Studio` IDE, you first have to provide your user name and previously generated PAT for artifactory to download packages for `Common` which is a part of this repository. When you load the solution, your IDE should request the credentials:  
```

Comment:
`Before you build or debug code in `Rider` or `Visual Studio` IDE, you first have to provide your user name and previously generated PAT for artifactory to download packages for `Common` which is a part of this repository.`

We don't have to pass username and password anymore, we can keep empty when packages are public. 

Please try on your computer additionally if this works, please remove 
"you first have to provide your user name and previously generated PAT for artifactory"

Reply:
I can't seem to recreate my previous problem, when afer removing the credentials, download was failing. However - without credentials I keep getting the login window. Have a look:
![request-login](https://github.com/evolutionary-architecture/evolutionary-architecture-by-example/assets/6518006/90b500af-66d8-4d80-bea4-c2f7edf6d0e6)

Reply:
Okay, so let's keep like is it.